AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -182,6 +183,15 @@ public class FullscreenMessage: NSObject, WKNavigationDelegate, FullscreenPresen
         }
     }
 
+    /// Delegate method invoked when the webView navigation is complete.
+    public func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        if navigation == self.loadingNavigation {
+            if let funcBlock = self.listener?.webViewDidFinishLoading {
```

Comment:
You should be able to just do `self.delegate?.webViewDidFinishLoading(webView: webView)`

Reply:
Yes this can be simplified since there is no logic for the else block (when optional protocol method is not available)